ArcGISRuntimeSDKQt_CppSamples/Analysis/AnalyzeHotspots/AnalyzeHotspots.cpp

```diff
@@ -31,10 +31,6 @@ AnalyzeHotspots::AnalyzeHotspots(QQuickItem* parent /* = nullptr */):
 {
 }
 
-AnalyzeHotspots::~AnalyzeHotspots()
-{
-}
-
```

Comment:
It doesn't save as many lines of code, but is it worth considering leaving these like this in the cpp files?

```cpp
AnalyzeHotspots::~AnalyzeHotspots() = default;
```

This has the benefit of:
1. No code is implemented in headers, even empty functions. Should help with build times.
2. Destructors implemented in the class require consumers of that class to know how to destruct all the members in some cases (especially any members that are std::unique_ptr). This can defeat forward declarations.

I don't think the second one affects us, so more just an academic observation. The first one may be a non-issue as well, doubtful build times will be affected.